_data/config.yml

```diff
@@ -1,3 +1,3 @@
-grpc_release_branch: v1.0.0
```

Comment:
So this value of this variable goes into link like this:

"https://github.com/grpc/grpc/blob/v1.2.0/examples/protos/route_guide.proto"

So as long as it can go after the `/blob/` part of the github URL that works. And tags work in that part of the URL. So yes the variable naming might be a bit off (both branch name and tag name can go after `/blob/`) but we can continue with this update.